cni/network/invoker_cns_test.go

```diff
@@ -1457,6 +1459,9 @@ func TestCNSIPAMInvoker_Add_SwiftV2(t *testing.T) {
 	ibMacAddress := "bc:9a:78:56:34:12"
 	ibParsedMacAddress, _ := net.ParseMAC(ibMacAddress)
 
+	accelnetAddress := "ab:cd:ef:12:34:56"
+	accelnetParsedMacAddress, _ := net.ParseMAC(ibMacAddress)
```

Comment:
Did you mean accelnetAddress here?
```suggestion
	accelnetParsedMacAddress, _ := net.ParseMAC(accelnetAddress)
```